gs_api_client/base/dispatch.py

```diff
@@ -41,6 +44,14 @@ def dispatch(self):
                 self._adapt_result_to_dict()
             return self._result
 
+    def _is_request_arg_contain_uuid(self):
+        for arg in self._request_args:
```

Comment:
why do u iterate over all args to check a request uuild!?

Reply:
The issue is actually not about request_uuids, it's about uuids in general. F.e. when getting a single server, you provide the server's `object_uuid`. Same with any other object. In case of subresources, you might provide multiple `object_uuid`s.